paperqa/contrib/zotero.py

```diff
@@ -240,7 +240,7 @@ def iterate(  # noqa: C901, PLR0912
             _pdfs = [self.get_pdf(item) for item in _items]
 
             # Filter:
-            for item, pdf in zip(_items, _pdfs):
+            for item, pdf in zip(_items, _pdfs, strict=False):
```

Comment:
Can you add a comment why we're not `strict` here? To me, it seems we should be `strict`

Reply:
black junk - fixed